src/Themes.scss

```diff
@@ -1462,19 +1476,47 @@ body.contrast-theme {
 
     .ProgramBlockEditor__character-position-button:not(.ProgramBlockEditor__character-position-button--disabled) {
```

Comment:
This rule is causing disabled reposition buttons to have no focus style for the contrast theme. We should specify the :not rule either inside of this rule, on a separate line.

Reply:
Good catch!  Fixed.